lib/db_utils.pm

```diff
@@ -0,0 +1,62 @@
+# Copyright (C) 2015-2019 SUSE LLC
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, see <http://www.gnu.org/licenses/>.
+
+package db_utils;
+
+use base Exporter;
+use Exporter;
```

Comment:
I'm not 100% sure, but I think this isn't needed if you use `use base Exporter`.
Just in case, this is how perldoc say the best way of exporting: https://perldoc.perl.org/Exporter.html#Playing-Safe